src/app/core/entity-components/entity-select/entity-select/entity-select.component.ts

```diff
@@ -38,16 +38,14 @@ export class EntitySelectComponent<E extends Entity> implements OnChanges {
   /**
    * The entity-type (e.g. 'Child', 'School', e.t.c.) to set.
    * @param type The ENTITY_TYPE of a Entity. This affects the entities which will be loaded and the component
-   *             that displays the entities.
+   *             that displays the entities. Can be an array giving multiple types.
    * @throws Error when `type` is not in the entity-map
    */
-  @Input() set entityType(type: string) {
-    this.loading.next(true);
-    this.entityMapperService.loadType<E>(type).then((entities) => {
-      this.allEntities = entities;
-      this.loading.next(false);
-      this.formControl.setValue(null);
-    });
+  @Input() set entityType(type: string | string[]) {
+    if (typeof type === "string") {
+      type = [type];
```

Comment:
Lets make `withPrefix` true when `type` is an array. I cannot imagine a scenario where this would not be wanted.

Reply:
good point, I am pulling this from the EditEntityArrayComponent down into the general EntitySelectComponent :+1: